Added Festive Moogles

kaincenteno opened the pull request:
Added scripts for Festive Moogles on each starter city.

TeoTwawki commented:
this is wrong. This only happens if other conditions are met - you have no conditions here.

these npc's also have ***multiple events*** most of which are trade events (mog pells of various colors).

kaincenteno commented:
@TeoTwawki  sure :cat:   i'll work on it today/tomorrow

kaincenteno commented:
@TeoTwawki  changes made:
* added python script for GM to give items
* changed festive moogle to use variables, added cs if no variable assigned.
* commented trade for festive moogles for someone else to tackle it 😸

TeoTwawki commented on scripts/zones/Port_Bastok/npcs/Festive_Moogle.lua:
```diff
@@ -5,17 +5,33 @@
-        [8] = function(player) return (player:getCharVar("festiveMoogleChocoboShurt")) end,
-    }
+function onTrade(player,npc,trade)
+    --TODO: trade of pellet for prize
```
its called a "pell". not pellet.

Green Mog Pell
Red Mog Pell
Silver Mog Pell
Gold Mog Pell

etc etc

kaincenteno replied:
got it, will rename it to pells =)

TeoTwawki commented on tools/festive_moogle/give_items.py:
```diff
@@ -0,0 +1,112 @@
+sqldatabase = "dspdb"       #
+#############################
+
+
```
why would you even do this? its not hard to create char vars

kaincenteno replied:
trying to be nice?
/shrug

kaincenteno replied:
I can erase it if thats the ask =)

TeoTwawki replied:
someone else can decide if project wants this. ***not it***

TeoTwawki replied:
@whasf I thrust the python thingy onto you.

TeoTwawki replied:
silence is acceptance merging

TeoTwawki commented on scripts/zones/Windurst_Walls/npcs/Festive_Moogle.lua:
```diff
@@ -5,30 +5,57 @@
+    if festiveItems[1] then
+        player:startEvent(380, unpack(festiveItems))
+    else
+        player:startEvent(381)
```
you've used the same csid at all 3 npc's. They do not use the same ID. the correct default event for the one in windurst is 504.

kaincenteno replied:
my bad. i copy and pasted it after making the changes, will fix this =)
